ally/find-food.php

```diff
@@ -30,7 +30,7 @@ function current() {
 				            global $phonenum;
 				            if (parent::key() == "Agency Name") {
 ?> 
-				<div class="col-sm-6">
+				<div class="col-sm-6" style="font-size: 130%; height: 46px; border-bottom: dashed gray 1px">
```

Comment:
So the best way to do this isn't with an inline style like this.  Instead you should add another class after col-sm-6 and then describe that class in the css file.